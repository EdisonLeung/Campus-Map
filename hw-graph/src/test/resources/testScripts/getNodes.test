# tests whether ListNodes works properly by adding a node, printing the node
# and then adding another one and seeing if ListNode will list the new node also

# Create graph
CreateGraph getNodes

# Add nodes
AddNode getNodes a

# List nodes
ListNodes getNodes

# Add another node
AddNode getNodes b

# List nodes
ListNodes getNodes