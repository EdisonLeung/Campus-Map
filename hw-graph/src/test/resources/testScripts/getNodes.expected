# tests whether ListNodes works properly by adding a node, printing the node
# and then adding another one and seeing if ListNode will list the new node also

# Create graph
created graph getNodes

# Add nodes
added node a to getNodes

# List nodes
getNodes contains: a

# Add another node
added node b to getNodes

# List nodes
getNodes contains: a b